guides/rails_guides/markdown.rb

```diff
@@ -95,7 +95,7 @@ def generate_header
 
       def generate_description
         sanitizer = Rails::Html::FullSanitizer.new
-        @description = sanitizer.sanitize(@header).squish
+        @description = sanitizer.sanitize(@header).squish.delete_prefix(@heading)
```

Comment:
Shouldn't the heading be in the description?

Reply:
Imo it should not be in the description.

E.g. here's a typical OG graph rendering of the ActionCable guide page before this change:
<img width="393" alt="image" src="https://github.com/user-attachments/assets/8425dfcd-fe9d-4bf8-b642-0675b9973dbf">

Or a rendered meta description in the Google SERPs:
<img width="605" alt="image" src="https://github.com/user-attachments/assets/04b73326-690d-4577-ad18-4216d6c6b624">

Most of the times, when the description is rendered, there's always the title visible as well, so I'd personally remove it in the description text itself 👍

Reply:
I would like @p8 or @AmandaPerino to chime in here but that makes sense to me.

Reply:
Also makes sense to me! As long as the main header does show up somewhere like the 2 examples above, it makes sense that we'd remove the duplication.

Reply:
Great! Thank you @p8 :)